File: app/src/main/java/com/arnaudpiroelle/manga/provider/japscan/api/JapScanDataApiService.java
package com.arnaudpiroelle.manga.provider.japscan.api;

import retrofit.client.Response;
import retrofit.http.GET;
import retrofit.http.Path;

public interface JapScanDataApiService {
    @GET("/lecture-en-ligne/{mangaAlias}/{chapterNumber}/{pageNumber}.{extension}")
    Response downloadPage(@Path("mangaAlias") String mangaAlias, @Path("chapterNumber") String chapterNumber,
                          @Path("pageNumber") String pageNumber, @Path("extension") String extension);
}

File: app/src/main/kotlin/com/arnaudpiroelle/manga/provider/japscan/api/JapScanDataApiService.kt
package com.arnaudpiroelle.manga.provider.japscan.api

import retrofit.client.Response
import retrofit.http.GET
import retrofit.http.Path

interface JapScanDataApiService {
    @GET("/cr-images/{mangaAlias}/{chapterNumber}/{pageNumber}.{extension}")
    fun downloadPage(@Path("mangaAlias") mangaAlias: String, @Path("chapterNumber") chapterNumber: String,
                     @Path("pageNumber") pageNumber: String, @Path("extension") extension: String): Response
}
